apps/accounts/utils/general.py

```diff
@@ -175,7 +180,7 @@ def get_user_by_email(email):
                 )
             except UserModel.MultipleObjectsReturned:
                 user = None
-                request = get_current_request()
+                request = get_current_request()  # noqa: F841
```

Comment:
I don't think F841 is the right one. Why?

I think below needs to get updated to:

`rollbar.report_exc_info(request=request)` ?

Let's double check against other places.